lookout/style/typos/benchmarks/typo_commits_report.py

```diff
@@ -174,8 +226,8 @@ def _finalize(self, reports: Iterable[Dict[str, str]]) -> Iterator[Dict[str, str
         scores.detection_recall = scores.detection_true_positive / (
             scores.detection_true_positive + scores.detection_false_negatives
         )
-        scores.fix_accuracy = scores.fix_accuracy / len(reports)
-        scores.top3_fix_accuracy = scores.top3_fix_accuracy / len(reports)
+        scores.fix_accuracy = scores.fix_accuracy / scores.detection_true_positive
+        scores.top3_fix_accuracy = scores.top3_fix_accuracy / scores.detection_true_positive
```

Comment:
this one is important. The number of correct fixes should be divided by the number of detection.